mp3info, zbd, sfill: add mnemonic hints and option placeholders (#16919)
tldr-pages/tldr: pages/common/zdb.md

`zdb`

> - Show detailed [C]onfiguration for a specific ZFS pool:
> 
> `zdb -C {{poolname}}`

Reviewer: ```suggestion
- Show detailed configuration for a specific ZFS pool:

`zdb {{[-C|--config]}} {{poolname}}`
```
This program supports long options so use option placeholders instead

> - Show statistics about number, size and deduplication of [b]locks:
> 
> `zdb -b {{poolname}}`

Reviewer: ```suggestion
- Show statistics about number, size and deduplication of [b]locks:

`zdb {{[-b|--block-stats]}} {{poolname}}`
```
Same here

- Show detailed configuration for a specific ZFS pool:

`zdb {{[-C|--config]}} {{poolname}}`

> - Show statistics about number, size and deduplication of [b]locks:

Reviewer: ```suggestion
- Show statistics about number, size and deduplication of blocks:
```
My bad, I forgot to remove the mnemonic
